templates/error/error.md

```diff
@@ -0,0 +1,13 @@
+# {{{ name }}}
```

Comment:
So you can actually do `<html>` and whatever tags you want in markdown. If using kramdown (if that still is a thing), you can then even continue using markdown syntax inside of those elements.

Reply:
But then you end up with a html/markdown hybrid which seems less prettier when used in non-html formats. That might also be specific for that library? Since I did look into a way to set something like a title in markdown and there seemed to be no general solution for this. The error descriptions would then also have to contain the entire html file since the title field would need to be in the head part, unless there's a library that takes care of that. At that point it seems to me that we lose a lot of the advantage of having the descriptions only in markdown and we might as well just write them in html directly?

Reply:
> html/markdown hybrid

HTML tags are subset of the markdown language, so it's technically markdown: https://daringfireball.net/projects/markdown/syntax#html

The title can just be the error code or so?

Reply:
Although I'm happy actually with the way things are now.

Idea for title: the markdown could contain a stray `<title>` tag that is hoisted out. (I do this on my site.)

Reply:
> Idea for title: the markdown could contain a stray `<title>` tag that is hoisted out. (I do this on my site.)

Yea can do that. It just means that the `MarkdownToHtmlConverter` needs to have some more logic to do this. Technically this could confuse people who for some reason have a markdown file starting with that title tag and did not expect this behaviour but I'm guessing that's not a large group